src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -584,6 +584,12 @@ private void CompileNamedType(NamedTypeSymbol containingType)
                 }
             }
 
+            var properties = members.OfType<SourcePropertySymbolBase>();
```

Comment:
>members.OfType<SourcePropertySymbolBase>() [](http://example.com/codeflow?start=29&length=42)

Does this produce an IEnumerable? We prefer avoiding going from an array to IEnumerable, or using Linqwhen a simple for/foreach loop over the original collection can be used to achieve the goal. This way some allocations can be avoided.